src/docfx/lib/JsonUtility.cs

```diff
@@ -213,6 +237,25 @@ private static void LogWarningForNullValue(JToken item, List<Error> errors, JTok
             }
         }
 
+        private static List<Error> RegisterLicense(string license)
```

Comment:
>List<Error> [](start = 23, length = 11)

just Error?

Reply:
Basically this method is used to call License.RegisterLicense, it could be void if we just throw the exception

---
In reply to: [201662287](https://github.com/dotnet/docfx/pull/3052#discussion_r201662287) [](ancestors = 201662287)